flax/linen/kw_only_dataclasses.py

```diff
@@ -122,7 +130,8 @@ def _process_class(cls, extra_fields=None, **kwargs):
   base_dataclass_fields = {}  # dict[cls, cls.__dataclass_fields__.copy()]
 
   # The keyword only fields from `cls` or any of its base classes.
-  kw_only_fields = {}  # dict[field_name, tuple[annotation, default]]
+  # dict[field_name, tuple[annotation, default]]
```

Comment:
This type in a comment feels a bit odd, now that `kw_only_fields` is annotated. Feel free to keep as is, but alternatively you could

* merge it into the human readable line above: "A mapping of keyword-only fields ... to a pair of ...";
* introduce type aliases, e.g. `Annotation = Any` etc;
* just drop it entirely :)

Reply:
Ok, did the future a favor and added type aliases.